hayeon/baekjoon/입출력연습/1000-A+B.js

```diff
@@ -0,0 +1,18 @@
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const input = [];
+
+rl.on("line", function (line) {
+  input.push(line);
+}).on("close", function () {
+  // 풀이
+  const inputData = input[0].split(" ");
+
+  const [a, b] = inputData.map(Number);
+
+  console.log(a + b);
```

Comment:
```suggestion
  const [a, b] = input[0].split(" ").map(Number);

  console.log(a + b);
```
- 입출력 잡을 때는 메서드 체이닝 최대한 이용해주셔도 좋습니다 :)

Reply:
감사합니다!😆 처음푸는 방식이라 어렵네요,,후